docs/users.rst

```diff
@@ -490,6 +490,16 @@ version
 This key is no longer used by OSBS and is only kept in the schema for backwards
 compatibility.
 
+
+set_release_env
+~~~~~~~~~~~~~~~
+
+Optional string.  If set, the image's Dockerfile will be amended with
```

Comment:
"amended with": can we say it will be added to the start of each stage in the Dockerfile, directly after FROM? (can be clarified in a separate paragraph if that makes it easier)

I think it's useful to be explicit about which stages we'll modify here.

Reply:
Clarified that we're editing all of them.